gammapy/maps/tests/test_hpxnd.py

```diff
@@ -505,4 +505,33 @@ def test_smooth(kernel):
         cutout.smooth(0.2 * u.deg, kernel)
 
     with pytest.raises(ValueError):
-        m_nest.smooth(0.2 * u.deg, "box")
\ No newline at end of file
+        m_nest.smooth(0.2 * u.deg, "box")
+
+def test_hpxmap_read_healpy(tmp_path):
+    path = tmp_path / "tmp.fits"
+
+    npix = 12 * 1024 * 1024
+    m = [np.arange(npix), np.arange(npix)-1, np.arange(npix)-2]
+    import healpy as hp
+    hp.write_map(
+        filename=path,
+        m=m, nest=False,
+        column_names=["data map", "background map", "exposure map"],
+        overwrite=True
+    )
+    from gammapy.maps.utils import find_bintable_hdu
```

Comment:
Please move those imports to the top of the file. Core dependencies such as Astropy, Numpy and Gammapy as well are just imported at the top of the file...